### PR TITLE
[SC] Added unsupported token standard

### PR DESCRIPTION
**Pull request by hyunsooda**
## Proposed changes

Added a description for unsupported token standard. 
Reference: https://github.com/klaytn/klaytn/pull/1445

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Minor Issues and Typos
- [ ] Major Content Contribution
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to reach out. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [ ] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn-docs/blob/master/CONTRIBUTING.md)
- [ ] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn-docs)
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large content contribution, kick off the discussion by explaining why you would suggest the content contribution, etc...

### CODE REVIEW

**Review by aidan-kwon — COMMENTED** [review]
Do you have plans to fill more content on `Bridge Contract` page?
If not, it would be better to put this content on [Value Transfer](node/service-chain/references/value-transfer.md) page

File: docs/node/service-chain/references/contract.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Bridge contract
+
+ServiceChain (SC) currently has three types of coin (`KLAY`, `ERC-20`, and `ERC-721`) transfer features between chains.
```
Comment (2dvorak) [inline docs/node/service-chain/references/contract.md:3]:
```suggestion
ServiceChain (SC) currently has three types of value (`KLAY`, `ERC-20`, and `ERC-721`) transfer features between chains.
```

Comment (hyunsooda) [reply docs/node/service-chain/references/contract.md]: Some of the contents were removed and some were moved to the other page. PTAL.

**Comment by hyunsooda** [comment]
@aidan-kwon, Okay. I pieced in this content to `reference/value-transfer.md` file.

File: docs/node/service-chain/references/contract.md (hunk 1 above)
Comment (2dvorak) [reply docs/node/service-chain/references/contract.md]: OK, thanks.

File: docs/node/service-chain/references/value-transfer.md [hunk 2]
```diff
@@ -284,6 +284,11 @@ The ERC-721 contract should implement the function, then.
 function requestValueTransfer(uint256 _uid, address _to) external
 ```
 
+### onERC721Received() <a id="unsupported-onERC721Received"></a>
+The ERC-721 standard has the [onERC721Received](https://eips.ethereum.org/EIPS/eip-721) callback function.
+The `onERC721Received()` works with `safeTransferFrom()` function, but the current bridge contract implementation uses `transferFrom()`, which menas the `onERC721Recieved()` is not expected to be called.
```
Comment (kjhman21) [inline docs/node/service-chain/references/value-transfer.md:289]:
```suggestion
The `onERC721Received()` works with `safeTransferFrom()` function, but the current bridge contract implementation uses `transferFrom()`, which means the `onERC721Recieved()` is not expected to be called.
```

**Comment by scott-klaytn** [comment]
Already incorporated into new Klaytn docs which is available after migration.

**Pull request closed without merging**